src/api/PDFPageOptions.ts

```diff
@@ -51,15 +51,19 @@ export interface PDFPageDrawSVGOptions {
   opacity?: number;
   borderColor?: Color;
   borderOpacity?: number;
+  dashArray?: number[];
+  dashPhase?: number; 
 }
 
 export interface PDFPageDrawLineOptions {
   start: { x: number; y: number };
   end: { x: number; y: number };
   thickness?: number;
   color?: Color;
-  lineCap?: LineCapStyle;
   opacity?: number;
+  lineCap?: LineCapStyle;
+  dashArray?: number[];
+  dashPhase?: number; 
```

Comment:
These **should** continue to be called `dashArray` and `dashPhase` (without the `border...` prefix), because they apply to the line itself, not its border.